### DISCUSSION TITLE
Adjust subtitle offset

### DISCUSSION THREAD

**Discussion by tremby** [jellyfin/jellyfin-android#839] [Q&A] [answered]
I saw here https://features.jellyfin.org/posts/178/synchronize-subtitles-timing-delay that in 10.4 the web client added the ability to adjust caption delay.

A comment on there suggests that it was also added to the Android client, but I can't figure out how to access that feature. Was the person who posted that mistaken, or am I missing something?

**Comment by Maxr1998** [comment, marked as answer]
It depends on which player you use. The web player is the same as the one in the web interface, and should support subtitle offsets already. The integrated player however doesn't support it yet.

**Reply by tremby** [reply]
Ah, thanks. I wasn't aware of there being different possible players to use in the Android client. I found the setting and sure enough I can change the subtitle offset when I switch to the web player.

What are the pros and cons of each?

**Reply by Maxr1998** [reply]
Integrated player is considered to be the future, since it has the best performance and supports the most codecs. However, a lot of features are still missing compared to the web player.
